undertow/src/main/java/org/wildfly/extension/undertow/filters/ModClusterDefinition.java

```diff
@@ -325,26 +335,72 @@ public HttpHandler createHttpHandler(Predicate predicate, ModelNode model, HttpH
 
     }
 
+    @Override
+    public void registerOperations(ManagementResourceRegistration parent) {
```

Comment:
Since your registering more than the operations here, I would suggest using a single registration method.

Reply:
OK